src/main.cc

```diff
@@ -491,15 +491,15 @@ int main( int argc, char ** argv )
 
       // TODO: Some `langauge`s in GD's ts uses - instead of _
       if ( loadTranslation_qlocale( *qt_ts, "qt", "_", QLibraryInfo::path( QLibraryInfo::TranslationsPath ) )
-           && qt_ts->language() == gd_ts->language().replace( '-', '_' ) ) {
+           && qt_ts->language() == gd_ts->language() ) {
         QCoreApplication::installTranslator( qt_ts );
       }
 
       if ( loadTranslation_qlocale( *webengine_ts,
                                     "qtwebengine",
                                     "_",
                                     QLibraryInfo::path( QLibraryInfo::TranslationsPath ) )
-           && webengine_ts->language() == gd_ts->language().replace( '-', '_' ) ) {
+           && webengine_ts->language() == gd_ts->language() ) {
```

Comment:
webengine_ts->language() == gd_ts->language() 

should be constraint the loaded translation file is the same as
`webengine_ts->load(qtwebengine_${language})`   ,is it?

Reply:
The check now can be deleted, but i am not 100% certain.

The issue was https://github.com/xiaoyifang/goldendict-ng/discussions/2117